Wikipedia/Code/ExploreViewController.swift

```diff
@@ -100,6 +100,12 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         stopMonitoringReachability()
         isGranularUpdatingEnabled = false
     }
+    
+    @objc private func databaseHousekeeperDidComplete() {
+        DispatchQueue.main.async {
+            self.refresh()
```

Comment:
I'm on occasion running into a situation where the content is deleted, but the explore feed doesn't refresh (see screenshot). I have English (primary) and Arabic in my languages, scroll several days worth of explore feed content, then perform a Settings > Clear cached data operation. I haven't been able to reliably reproduce this though yet. Certainly not a dealbreaker as pulling to refresh the explore feed still refreshes the feed, but just wanted to check in to see if you'd encountered anything like this yourself.
![sim](https://github.com/wikimedia/wikipedia-ios/assets/5652327/4d372c53-fc2e-4892-9ea1-295ebb15398c)